### PR TITLE
feat(demo2): add destroy mode to celebrate commit 1000!

### PR DESCRIPTION
**Pull request by joshka**
```shell
cargo run --example demo2 --features="crossterm widget-calendar"
```

Press `d` to activate destroy mode and Enjoy!

Destroy Demo2

Vendors a copy of tui-big-text to allow us to use it in the demo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,8 @@ Badge]](./LICENSE)<br>
 [![Codecov Badge]][Codecov] [![Deps.rs Badge]][Deps.rs] [![Discord Badge]][Discord Server]
 [![Matrix Badge]][Matrix]<br>
 
-[Ratatui Website] · [API Docs] · [Examples]<br>
-[Contributing] · [Changelog] · [Breaking Changes]<br>
-[Report a bug] · [Request a Feature] · [Create a Pull Request]
+[Ratatui Website] · [API Docs] · [Examples] · [Changelog] · [Breaking Changes]<br>
+[Contributing] · [Report a bug] · [Request a Feature] · [Create a Pull Request]
```
Comment (Valentin271) [inline README.md:35]: Is this normal?

Comment (joshka) [reply README.md:35]: Yep. We generate the Readme from lib.rs using cargo-rdme. This was updated and the readme wasn't generated with the new changes. Updating the image updated this automatically.

File: examples/demo2/app.rs [hunk 2]
```diff
@@ -82,10 +103,142 @@ impl App {
...
     }
+
+    fn should_quit(&self) -> bool {
+        self.mode == Mode::Quit
+    }
+}
+
+/// delay the start of the animation so it doesn't start immediately
+const DELAY: usize = 240;
+/// higher is means more pixels per frame are modified in the animation
```
Comment (Valentin271) [inline examples/demo2/app.rs:121]:
```suggestion
/// higher means more pixels per frame are modified in the animation
```

File: README.md (hunk 1 above)
Comment (Valentin271) [reply README.md:35]: Alright

File: examples/demo2/app.rs (hunk 2 above)
Comment (joshka) [reply examples/demo2/app.rs]: Addressd in https://github.com/ratatui-org/ratatui/pull/809/commits/5337d0b74d3861624f16d685b7decd8c1b4f2397

**Review by Valentin271 — APPROVED** [review]
:tada:

**Pull request merged**